projects/ngrx.io/content/guide/effects/lifecycle.md

```diff
@@ -91,6 +91,57 @@ export class AuthEffects {
 }
 </code-example>
 
+### Customizing the Effects Error Handler
+Starting with version 9 the behavior of the default resubscription handler can be customized 
+by providing a custom handler using the `EFFECTS_ERROR_HANDLER` injection token.
+
+This will allow you to provide your own custom behavior such as only retrying on
+certain "retryable" errors, or retrying a set number of times etc.
+
+<code-example header="customise-error-handler.effects.ts">
+```ts
+import { EffectsModule, EFFECTS_ERROR_HANDLER } from '@ngrx/effects';
+import { MovieEffects } from './effects/movie.effects';
+import { CustomErrorHandler, isRetryable } from '../custom-error-handler';
+import { Action } from '@ngrx/store';
+import { Observable, throwError } from 'rxjs';
+import { retryWhen, mergeMap } from 'rxjs/operators';
+
+export function effectResubscriptionHandler<T extends Action>(
+  observable$: Observable<T>,
+  errorHandler?: CustomErrorHandler
+): Observable<T> {
+  return observable$.pipe(
+    retryWhen(errors =>
+      errors.pipe(
+        mergeMap(e => {
+          if (isRetryable(e)) {
+            return errorHandler.handleRetryableError(e);
+          }
+
+          errorHandler.handleError(e);
+          return throwError(e);
+        })
+      )
+    )
+  );
+}
+
+@NgModule({
+  imports: [EffectsModule.forRoot([MovieEffects])],
+  providers: [
+    {
+      provide: EFFECTS_ERROR_HANDLER,
+      useValue: effectResubscriptionHandler,
+    },
+    {
+      provide: ErrorHandle, 
```

Comment:
Oops! This is when I should plug my library [embedme](https://github.com/zakhenry/embedme) which allows for markdown files to import typescript files directly, ensuring mistakes like this can't slip through, plus ensures breaking changes in documentation code is caught. 😁